.env.sample

```
@@ -3,7 +3,9 @@ GATSBY_ALGOLIA_SEARCH_KEY=
 ALGOLIA_ADMIN_KEY=
 NODE_ENV=development
 ALGOLIA_SKIP_INDEXING=true
+GITHUB_TOKEN=
 CONTENTFUL_ACCESS_TOKEN=
+ROADIE_API_TOKEN=
```

This was needed but failed silently.